Makefile

```diff
@@ -74,6 +74,7 @@ endif
 	$(call php-0, apk add --no-cache graphicsmagick tzdata $(ADD_PHP_EXT))
 	# Set up timezone
 	$(call php-0, cp /usr/share/zoneinfo/Europe/Paris /etc/localtime)
+	make -s newrelic
```

Comment:
Maybe best to not add in make all by default ? cause newrelic requires registration etc

Reply:
Added comment - it will work only in `NEW_RELIC_LICENSE_KEY=xxx make provision` will be fired or when in CI environment variable defined

Reply:
So NEW_RELIC_LICENSE_KEY should be added to .env.default (or at least documented in README.md)

Reply:
Document ++ because it needed only on RA
For local we have xhprof and blackfire

Reply:
Fixed by changing readme bellow, moreover sharing a key/secret via `.env` was initially bad idea